WordPress/WordPressTest/PostRepositoryPostsListTests.swift

```diff
@@ -63,7 +63,7 @@ class PostRepositoryPostsListTests: CoreDataTestCase {
     }
 
     func testFetchingAllPages() async throws {
-        try await preparePostsList(type: "page", total: 1_120)
+        stubGetPostsList(type: "page", total: 1_120)
```

Comment:
This test takes ~7s to run in CI, according to [Test Analytics](https://buildkite.com/organizations/automattic/analytics/suites/wordpress-ios/runs/325ce80a-8caf-4e29-aba3-612b72517cb9). The 1,120 amount was there already, but I wonder if the value we get from it is worth the speed bump.

![image](https://github.com/wordpress-mobile/WordPress-iOS/assets/1218433/fac129f2-fdc9-4b70-a70e-e71cbd87a588)

Reply:
I have removed it in https://github.com/wordpress-mobile/WordPress-iOS/pull/21997/commits/cfe7a675ba31ce724a3b180bb4420df87537730f. It's duplicated with other tests.